README.md

```diff
@@ -88,108 +75,172 @@ By default, entries in the OpenAPI Explorer pane are sorted alphabetically. If y
+Many of the issues reported by 42Crunch Audit have fixes associated with them. These are code snippets that you can insert into the OpenAPI file and then customize with the appropriate value.
 
-### Navigate the issues in the audit report
+1. Cick some of the error lines in your OpenAPI file to bring up the QuickFix blue icon on the left of the line.
+2. Click the Quickfix icon and select the fix to apply.
+3. Tweak the values in the inserted code snippet as you see fit.
 
-After the audit finishes, you get the audit report directly in the VS Code view, side by side with your code. Depending on your API definition, the report might be long, so here are some handy ways to navigate the found issues.
+![Quick Fixes in VSCode](./images/Quick_Fixes_in_VSCode.gif)
 
-1. To scroll through all issues and their details, use the text panel on the right.
+## (New!) Dynamic API Security testing
 
-![Perform REST API Security Audit](https://github.com/42Crunch/vscode-openapi/blob/master/images/Perform%20REST%20API%20Security%20Audit.gif?raw=true)
+42Crunch Audit performs a security analysis that does not require any live API, just the definition itself. 42Crunch Scan leverages the OpenAPI definition to:
 
-2. Hover on an element that is underlined or marked with three dots (recommendations) in your code to see what the issues in that spot are. If you click on view the descriptions of only these issues, the rest of details are filtered out from the text panel, so you have less to scroll through.
+- Test the resilience and behavior of APIs by automatically generating security tests from the APIs' OpenAPI definition. Scan reproduces the typical behavior of a hacker by injecting bad payloads, bad tokens, and using invalid HTTP verbs and paths. This helps detect vulnerabilities early in the API life cycle, especially those associated with the [OWASP API Security Top 10](https://apisecurity.io/owasp-api-security-top-10/owasp-api-security-top-10-project/).
+- Validate that the implementation of the API conforms to its established contract: Scan checks all responses against the OpenAPI definition and detects unexpected responses and data leaks.
 
-![Details for specific issues](https://github.com/42Crunch/vscode-openapi/blob/master/images/Details%20for%20specific%20issues.gif?raw=true)
+![](https://img.shields.io/badge/Learning%20More!-Blue) Watch this [video](https://42crunch.com/free-user-faq/?utm_campaign=IDE&utm_medium=referral&utm_source=vscode#Free-API-Security-Testing-FAQs-4) to learn more about 42Crunch Scan.
 
-3. For a quick overall look, check the counts in the Status Bar. The different icons match the severity of the issue:
+APIs which thoroughly enforce compliance to an established contract are far more resilient to all types of attacks.
 
-- Error: critical or high
-- Warning: medium
-- Info: low
+![](https://img.shields.io/badge/Important-red)  You should only use 42Crunch Scan against APIs that you own, but not those of third parties.
```

Comment:
"You must only use 42Crunch Scan against APIs that you own, not those of third parties." to give this more weight and less hesitancy (the "but" doesn't fit in well).